src/Classes/Controllers/SaveApplicantController.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Portal\Controllers;
+
+use Portal\Models\ApplicantModel;
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+class SaveApplicantController
+{
+    private $applicantModel;
+
+    /**
+     * Instantiates SaveApplicantController
+     *
+     * @param ApplicantModel $applicantModel userModel object dependency
+     */
+    public function __construct(ApplicantModel $applicantModel)
+    {
+        $this->applicantModel = $applicantModel;
+    }
+
+    /**
+     * If user is logged in, invoke gets data from new applicant form and passes into insertNewApplicantToDb
+     * function for inserting into database.
+     * If successful Insert returns success true with message of application saved.
+     * 
+     * @param Request $request
+     * @param Response $response
+     * 
+     * @return Response
+     */
+    function __invoke(Request $request, Response $response)
+    {
+        if ($_SESSION['loggedIn'] === true) {
+            $data = ['success' => false, 'msg' => 'Application not saved', 'data' => []];
+            $statusCode = 401;
+
+            $newApplicationData = $request->getParsedBody();
+            $validatedApplicationData = [
+                'name'           => filter_var($newApplicationData['name'], FILTER_SANITIZE_STRING),
+                'email'          => filter_var($newApplicationData['email'], FILTER_SANITIZE_STRING),
```

Comment:
doesnt seem the email is being validated anywhere? this is only sanitizing. not validating. look up:
`FILTER_VALIDATE_EMAIL`